src/Resources/config/services/factory/fraud_suspicion.xml

```diff
@@ -4,9 +4,10 @@
         xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd"
 >
     <services>
+
         <service
-            id="bitbag.sylius_blacklist_plugin.factory.fraud_suspicion"
```

Comment:
As mention in [doc](https://github.com/Sylius/SyliusResourceBundle/blob/master/docs/services.md), default id will be `bitbag_sylius_blacklist_plugin.factory.fraud_suspicion`, so I create alias with `.` because everywhere is used with `.`(I didn't found better idea how to do it).